service/controller.go

```diff
@@ -20,12 +20,14 @@ import (
 	"net/http"
 	"strconv"
 	"strings"
+	"sync"
```

Comment:
Can we modify Copyright years for all files that we are modifying as part of this PR?

Reply:
Done